x-pack/plugins/monitoring/public/application/pages/page_template.tsx

```diff
@@ -54,7 +58,12 @@ export const PageTemplate: React.FC<PageTemplateProps> = ({
   }, [getPageData, currentTimerange]);
 
   const onRefresh = () => {
-    getPageData?.().catch((err) => {
+    const requests = [getPageData?.()];
+    if (isSetupModeFeatureEnabled(SetupModeFeature.MetricbeatMigration)) {
+      requests.push(updateSetupModeData());
+    }
+
+    Promise.allSettled(requests).then((results) => {
```

Comment:
I think this should be `.catch` because if I'm not missing anything here we don't have to do anything with results, we only want to process errors. (Although I'm working on the errors page, so I can also change it after you merge it to master)

Reply:
[allSettled](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) never rejects. Thought it would be appropriate here since requests don't look related and individual error handling may be a good option, but I'll leave it for the tasks dedicated to that :)